content/guide/07-element-directives.md

```diff
@@ -387,4 +387,173 @@ export default {
 {
 	test: ""
 }
-```
\ No newline at end of file
+```
+
+### Actions
+
+Components can get you far, but there are occasions when you need functionality for an element and wrapping it in a
+component prevents other functionality being added. How do you add a tooltip to the `<Link>` component from your
+favorite routing library?
+
+Actions let you decorate elements with additional functionality.
+
+```html
+<!-- { title: 'Actions' } -->
+Copy link: <input type=text value={url} use:autoselect>
```

Comment:
I wonder if we should use a different example? This can be achieved with

```html
<input on:focus="this.select()">
```

so may not be the most compelling demo